lib/handlers.js

```diff
@@ -180,7 +180,7 @@ function is_array(val) {
 
 function is_object(val) {
   // is this a non-array, non-null object?
-  return val && !(val instanceof Array) && typeof val === "object";
+  return val && !Array.isArray(val) && typeof val === "object";
```

Comment:
You'll want to also bump the version and add to the changes right?

Reply:
Good point, forgot we are not going to get the patch automatically. Will fix.